modules/core/src/main/java/org/apache/ignite/internal/commandline/TracingConfigurationCommand.java

```diff
@@ -0,0 +1,345 @@
+            optional(TracingConfigurationCommandArg.SCOPE.argName(), join("|", Scope.values())));
+
+        Command.usage(
+            log,
+            "Reset specific tracing configuration to the default. If both " +
+                TracingConfigurationCommandArg.SCOPE.argName() + " and " +
+                TracingConfigurationCommandArg.LABEL.argName() + " are specified then remove given configuration," +
+                " if only " + TracingConfigurationCommandArg.SCOPE.argName() +
+                " is specified then reset given configuration to the default." +
+                " Print reseted configuration.",
+            TRACING_CONFIGURATION,
+            TracingConfigurationSubcommand.RESET.text(),
+            grouped(TracingConfigurationCommandArg.SCOPE.argName(), join("|", Scope.values())),
+            optional(TracingConfigurationCommandArg.LABEL.argName()));
+
+        Command.usage(
+            log,
+            "Set new tracing configuration. If both " +
+                TracingConfigurationCommandArg.SCOPE.argName() + " and " +
+                TracingConfigurationCommandArg.LABEL.argName() + " are specified then add or override label" +
+                " specific configuration, if only " + TracingConfigurationCommandArg.SCOPE.argName() +
+                " is specified, then override scope specific configuration. Print applied configuration.",
+            TRACING_CONFIGURATION,
+            TracingConfigurationSubcommand.SET.text(),
+            grouped(TracingConfigurationCommandArg.SCOPE.argName(), join("|", Scope.values()),
+            optional(TracingConfigurationCommandArg.LABEL.argName()),
+            optional(TracingConfigurationCommandArg.SAMPLING_RATE.argName(),
+                "Decimal value between 0 and 1, " +
+                "where 0 means never and 1 means always. " +
+                "More or less reflects the probability of sampling specific trace."),
+            optional(TracingConfigurationCommandArg.INCLUDED_SCOPES.argName(),
+                "Set of scopes with comma as separator ",
+                join("|", Scope.values()))));
+    }
+
+    /**
+     * Execute tracing-configuration command.
+     *
+     * @param clientCfg Client configuration.
+     * @throws Exception If failed to execute tracing-configuration action.
+     */
+    @Override public Object execute(GridClientConfiguration clientCfg, Logger log) throws Exception {
+        if (experimentalEnabled()) {
+            try (GridClient client = Command.startClient(clientCfg)) {
+                UUID crdId = client.compute()
+                    //Only non client node can be coordinator.
+                    .nodes(node -> !node.isClient())
+                    .stream()
+                    .min(Comparator.comparingLong(GridClientNode::order))
+                    .map(GridClientNode::nodeId)
+                    .orElse(null);
+
+                VisorTracingConfigurationTaskResult res = executeTaskByNameOnNode(
```

Comment:
Perhaps, if you want to execute this task on a coordinator node, you can use `VisorCoordinatorNodeTask`.

Reply:
It's actually possible to execute this task on any server node.